### PR TITLE
Feat: French Translation

### PR DESCRIPTION
**Pull request by CMSEv7**
Work in progress, I am taking a pause and I shall continue later. I am at line 100.

### CODE REVIEW

**Comment by vos6434** [comment]
Make a Draft as it is not finished.

File: src/main/resources/assets/galacticraft/lang/fr_ca.json [hunk 1]
```diff
@@ -0,0 +1,510 @@
+{
+  "itemGroup.galacticraft.blocks": "Blocs de Galacticraft",
+  "itemGroup.galacticraft.items": "Items de Galacticraft",
+  "itemGroup.galacticraft.machines": "Machines de Galacticraft",
+  "block.galacticraft.silicon_ore": "Minerai de silicium",
+  "block.galacticraft.desh_ore": "Minerai de Desh",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/fr_ca.json:6]: Desh -> desh (consistency)

File: src/main/resources/assets/galacticraft/lang/fr_ca.json [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.mars_surface_rock": "Pierre de surface de Mars",
+  "block.galacticraft.mars_sub_surface_rock": "Pierre subsurface de Mars",
+  "block.galacticraft.mars_stone": "Pierre martienne",
+  "block.galacticraft.mars_cobblestone": "Cobblestone de Mars",
+  "block.galacticraft.mars_cobblestone_stairs": "Escaliers en cobblestone de Mars",
+  "block.galacticraft.mars_cobblestone_slab": "Dalle de cobblestone de Mars",
+  "block.galacticraft.mars_cobblestone_wall": "Muret de cobblestone de Mars",
+  "block.galacticraft.mars_copper_ore": "Minerai de cuivre",
+  "block.galacticraft.mars_tin_ore": "Minerai d'étain",
+  "block.galacticraft.mars_iron_ore": "Minerai de fer",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/fr_ca.json:43]: "... de Mars"

File: src/main/resources/assets/galacticraft/lang/fr_ca.json [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.moon_basalt_slab": "Dalle de basalte lunaire",
+  "block.galacticraft.moon_basalt_stairs": "Escaliers de basalte lunaire",
+  "block.galacticraft.moon_basalt_wall": "Muret de basalte lunaire",
+  "block.galacticraft.moon_rock_stairs": "Escalie5rs de pierre lunaire",
+  "block.galacticraft.moon_rock_slab": "Dalle de pierre lunaire",
+  "block.galacticraft.moon_rock_wall": "Muret de pierre lunaire",
+  "block.galacticraft.moon_dirt": "Terre lunaire",
+  "block.galacticraft.moon_copper_ore": "Minerai de cuivre",
+  "block.galacticraft.moon_tin_ore": "Minerai d'étain",
+  "block.galacticraft.moon_cheese_ore": "Minerai de fromage",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/fr_ca.json:27]: "... lunaire"

**Review by martmists-gh — CHANGES_REQUESTED** [review]
Decided to suggest some small changes while this is still WIP. 
tl;dr try to be consistent with the english (not leaving out any information) while keeping the translations in the french crowdin format ([link](https://crowdin.com/translate/minecraft/9435/enus-fr?filter=basic&value=0))

File: src/main/resources/assets/galacticraft/lang/fr_ca.json (hunk 1 above)
Comment (CMSEv7) [reply src/main/resources/assets/galacticraft/lang/fr_ca.json:6]: Desh est un nom propre, il a été inventé pour le _purpose_ de Galacticraft.

Comment (CMSEv7) [reply src/main/resources/assets/galacticraft/lang/fr_ca.json:6]: Desh est un nom propre, car il a été inventé pour Galacticraft.

**Comment by Yexiaowei11** [comment]
the Chinese translation
